src/text-fragment-utils.js

```diff
@@ -481,7 +481,7 @@ export const scrollElementIntoView = (element) => {
  * the node's ancestors so even if the node is visible according to its style
  * it might not be visible on the page if one of its ancestors is not visible.
  * @param {Node} node - the Node to evaluate
- * @returns {Boolean} - true if the node is visible. A node will be visible if
+ * @return {Boolean} - true if the node is visible. A node will be visible if
```

Comment:
Do we use @return instead of @returns? It seems like @returns is the standard jsdoc tag and @return is a synonym. https://jsdoc.app/tags-returns.html. VS Code code completion inserts @returns.

Reply:
I didn't do this manually. I don't really care about the distinction one way or another, but `clang-format --style=Google` seems to prefer `@return` and I vote for letting the machine do its thing.

Reply:
Hmm, maybe I have the wrong culprit then. Does it change when you run `npm install`?

Reply:
Yes, it's `npx eslint . --ext .js,.mjs --fix --ignore-pattern dist/`